### PR TITLE
Google Maps

### PR DESCRIPTION
**Pull request by kjrcda**
Working map up and running. Needs functionality.

### CODE REVIEW

File: webapp/views/partials/map.jade [hunk 1]
```diff
@@ -0,0 +1,8 @@
+div(class='container')
+    h1 Pick a Place on the Map
+
+    //this should actually be "div(ui-view)" but for some reason that doesn't work...
+    //See http://goo.gl/TVBKXO
```
Comment (paymog) [inline webapp/views/partials/map.jade]: These comments should be removed since you removed the `ui-view` tag.

**Comment by paymog** [comment]
`bower.json` hasn't been updated to include `lodash` or the `angularmaps......` packages.

File: webapp/views/layout.jade [hunk 2]
```diff
@@ -13,6 +13,9 @@ html(lang='en', ng-app='lunchr')
         script(src='bower_components/angular-ui-router/release/angular-ui-router.js')
         script(src='javascripts/angular-app.js')
         script(src='javascripts/controllers.js')
+        script(src='bower_components/lodash/dist/lodash.js')
```
Comment (paymog) [inline webapp/views/layout.jade:16]: why do we need this?

Comment (kjrcda) [inline webapp/views/layout.jade:16]: I am not 100% sure, I can look into if we need it or not. When I installed the framework lodash came with it. I think the map depends on it.

Comment (paymog) [inline webapp/views/layout.jade:16]: What happens if you comment this out?

**Comment by paymog** [comment]
Write tests for the maps controller

File: webapp/views/layout.jade (hunk 2 above)
Comment (kjrcda) [inline webapp/views/layout.jade:16]: It breaks. So yes we do need it.

**Comment by kyleleung75** [comment]
Heyo, we are going to close this until we have tests for the map stuff and because of the commits brandon made today

**Comment by kjrcda** [comment]
Why close it though? we could just keep it open so we dont forget about it?

**Comment by kyleleung75** [comment]
Just so that it is very obvious we aren't going to merge it yet. It can be
reopened can't it?

**Comment by kjrcda** [comment]
It can ya. Personally I find the PR help so people can see and review your work.

**Pull request closed without merging**